src/hotspot/share/gc/shenandoah/shenandoahCollectorPolicy.cpp

```diff
@@ -109,6 +95,17 @@ bool ShenandoahCollectorPolicy::is_at_shutdown() {
   return _in_shutdown.is_set();
 }
 
+bool is_explicit_gc(GCCause::Cause cause) {
```

Comment:
There is `ShenandoahControlThread::is_explicit_gc` too. I think we should have a shared definition for this somewhere. Maybe `ShenandoahControlThread` should delegate to `ShenandoahCollectorPolicy` then?